### DISCUSSION TITLE
在config dev.js文件中devServer的historyApiFallback进行配置不生效?

### DISCUSSION THREAD

**Discussion by zouchao1** [NervJS/taro#15254] [Q & A]
### 这个特性解决了什么问题？
taro源码中recursiveMerge对开发者customDevServerOption配置进行了一个merge，导致都是优先匹配    from: /./规则？

### 这个 API 长什么样？
merge完最后在加上from: /./规则

**Comment by ZakaryCode** [comment]
可以用 [webpackChain](https://nervjs.github.io/taro-docs/docs/config-detail/#h5webpackchain) 调整哈

**Reply by zouchao1** [reply]
能请教一下怎么写吗

**Reply by zouchao1** [reply]
应该是merge后我才操作吧，现在是我先执行webpackChain里面逻辑，才进行的merge吧
